src/components/v2/values/chartValuesDiff/ChartValuesView.tsx

```diff
@@ -890,6 +898,12 @@ const ChartValuesView = ({
                     deploymentAppType: isVirtualEnvironmentOnSelector
                         ? DeploymentAppTypes.MANIFEST_DOWNLOAD
                         : commonState.deploymentAppType,
+                    gitRepoURL:
```

Comment:
Please simplify this condition to avoid nested ternary. Use const for 'Default'

Reply:
Have simplified logic in refactor branch